prov/efa/src/efa.h

```diff
@@ -192,10 +192,15 @@ struct efa_cq {
 	efa_cq_read_entry	read_entry;
 	ofi_spin_t		lock;
 	struct ofi_bufpool	*wce_pool;
+	uint32_t	comp_mask;
 
 	struct ibv_cq_ex	*ibv_cq_ex;
 };
 
+enum {
+	EFA_CQ_SUPPORT_EFADV_CQ = 1 << 0,
```

Comment:
should be a macro, not enum

Reply:
I'm curious about the difference? I see rdma-core prefers to use enum. Is it a convention to use macro in libfabric?

Reply:
I think enum is for a list of constants that increases numerically 1 by 1.

But this one is being used as a bit mask. so should be a macro.

Also please use `BIT_ULL` to define it.

Reply:
Will address. I was referring to examples like [this](https://github.com/linux-rdma/rdma-core/blob/master/providers/efa/efadv.h#L38-L42) in rdma-core which uses enum instead.

Reply:
Addressed